### PR TITLE
[Feature] Add Volcano Helm Chart to terraform-aws-eks-data-addons #304

### PR DESCRIPTION
**Pull request by elygerou**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### Motivation
Would like to contribute and potentially present my contribution with a pattern in the container AoD be considered for Container TFC

Discussion with Vara Bothu as a first step for continuing on issue 284
### More

- [x] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ ] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### Additional Notes
SF_Data-on-EKS_Repos/01.repos - (main) > pre-commit run --all-files
trim trailing whitespace.................................................Passed
fix end of files.........................................................Passed
check for merge conflicts................................................Passed
detect private key.......................................................Passed
detect aws credentials...................................................Passed
Terraform fmt............................................................Passed
Terraform docs...........................................................Passed
Terraform validate with tflint...........................................Passed
Terraform validate.......................................................Passed

In Line 226 of the addon.tf file I have tried with the local folder; source = “/…/terraform-aws-eks-data-addons" and terraform plan was successful

End of terraform plan message:

Plan: 144 to add, 0 to change, 0 to destroy.

Changes to Outputs:
  + configure_kubectl = "aws eks --region us-west-2 update-kubeconfig --name trainium-inferentia"

No

### CODE REVIEW

File: ai-ml/trainium-inferentia/addons.tf [hunk 1]
```diff
@@ -247,21 +250,21 @@ resource "kubectl_manifest" "torchx_etcd" {
...
-data "http" "volcano_development_yaml" {
-  url = "https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml"
-}
+# #---------------------------------------------------------------
+# # Volcano Schduler for TorchX
+# # NOTE: This will be replaced with Helm Chart deployment with eks_data_addons
+# #---------------------------------------------------------------
+# data "http" "volcano_development_yaml" {
+#   url = "https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml"
+# }
```
Comment (vara-bonthu) [inline ai-ml/trainium-inferentia/addons.tf:259]: remove all commented lines

File: ai-ml/trainium-inferentia/addons.tf [hunk 2]
```diff
@@ -247,21 +250,21 @@ resource "kubectl_manifest" "torchx_etcd" {
...
 
-resource "kubectl_manifest" "volcano" {
-  yaml_body = <<-YAML
-    ${data.http.volcano_development_yaml.response_body}
-  YAML
+# resource "kubectl_manifest" "volcano" {
```
Comment (vara-bonthu) [inline ai-ml/trainium-inferentia/addons.tf:261]: remove all commented lines

File: ai-ml/trainium-inferentia/addons.tf [hunk 3]
```diff
@@ -247,21 +250,21 @@ resource "kubectl_manifest" "torchx_etcd" {
...
+#   yaml_body = <<-YAML
+#     ${data.http.volcano_development_yaml.response_body}
+#   YAML
 
-  depends_on = [module.eks.eks_cluster_id]
-}
+#   depends_on = [module.eks.eks_cluster_id]
+# }
```
Comment (vara-bonthu) [inline ai-ml/trainium-inferentia/addons.tf:267]: remove commented lines

**Review by vara-bonthu — COMMENTED** [review]
exclude these files and sub module directory from your PR image

and Only add the files that you have updated.

File: ai-ml/trainium-inferentia/addons.tf (hunk 1 above)
Comment (elygerou) [reply ai-ml/trainium-inferentia/addons.tf]: done

File: ai-ml/trainium-inferentia/addons.tf (hunk 2 above)
Comment (elygerou) [reply ai-ml/trainium-inferentia/addons.tf]: done

File: ai-ml/trainium-inferentia/addons.tf (hunk 3 above)
Comment (elygerou) [reply ai-ml/trainium-inferentia/addons.tf]: done as well, i am sorry I had to do git add . to remove all the wrong subfolders but wont happen again should be ok now

**Pull request closed without merging**